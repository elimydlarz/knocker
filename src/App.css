.App {
  padding: 20px;
}

input[type="text"] {
  width: 100%;
}

textarea {
  display: block;
  width: 100%;
  height: 100px;
}
