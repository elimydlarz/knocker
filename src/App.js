import React, { Component } from 'react';
import '../node_modules/skeleton-css/css/normalize.css';
import '../node_modules/skeleton-css/css/skeleton.css';
import './App.css';

import Knocked from './components/Knocked'

export default class App extends Component {
  render() {
    return (
      <div className="App">
        <h1>Knocker</h1>
        <Knocked />
      </div>
    );
  }
}
