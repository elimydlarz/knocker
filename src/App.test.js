jest.mock('./services/Window');

import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
  it('does not explode', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
