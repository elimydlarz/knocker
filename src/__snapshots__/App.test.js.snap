exports[`App does not explode 1`] = `
<div
  className="App">
  <h1>
    Knocker
  </h1>
  <div
    className="FlashMessage">
    So, tell me a bit about yourself...
  </div>
  <div
    className="Login">
    <form>
      <div
        className="TextInput">
        <input
          name="user"
          onChange={[Function bound updateUser]}
          placeholder="Who\'s knocking?"
          type="text"
          value="" />
      </div>
      <input
        name="password"
        onChange={[Function bound updatePassword]}
        placeholder="What\'s your password?"
        type="password"
        value="" />
      <div
        className="Submit">
        <input
          disabled={true}
          onClick={[Function bound transmit]}
          type="submit" />
      </div>
    </form>
  </div>
</div>
`;
